test/uk/gov/hmrc/tradergoodsprofilesrouter/service/DateTimeServiceSpec.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tradergoodsprofilesrouter.service
+
+import org.scalatestplus.play.PlaySpec
+import uk.gov.hmrc.tradergoodsprofilesrouter.service.DateTimeService.DateTimeFormat
+
+import java.time.Instant
+
+class DateTimeServiceSpec extends PlaySpec {
```

Comment:
missing test case for as StringHttp format. Also I would remove the toStringInMillis from the class and remove the test as this function is not used anywhere